libs/ledger-live-common/src/icons/providers/providers.ts

```diff
@@ -0,0 +1,12 @@
+import { CDN_URL } from "../consts/urls";
+
+type GetProviderIconUrlProps = {
+  cdn?: string;
+  name: string;
+  boxed: boolean;
```

Comment:
could this param be optional?

Reply:
If I change `boxed` in `getProviderIconUrl` to optional. Can I remove the default value in the `ProviderIcon/index.tsx` in both LLD & LLM?